### PR TITLE
Add option to disable canonical check when adding a schema version

### PR DESCRIPTION
**Pull request by joylyn**

### CODE REVIEW

File: webservice/src/main/resources/app/scripts/containers/Registry-Services/SchemaRegistry/SchemaVersionForm.jsx [hunk 1]
```diff
@@ -52,7 +52,8 @@ export default class SchemaVersionForm extends Component {
       showError: false,
       changedFields: [],
       showCodemirror: true,
-      schemaTextCompatibility: statusCode.Ok
+      schemaTextCompatibility: statusCode.Ok,
+      canonicalCheck: false
```
Comment (raju-saravanan) [inline webservice/src/main/resources/app/scripts/containers/Registry-Services/SchemaRegistry/SchemaVersionForm.jsx]: This should be called disableCanonicalCheck

File: webservice/src/main/resources/app/scripts/containers/Registry-Services/SchemaRegistry/SchemaVersionForm.jsx [hunk 2]
```diff
@@ -116,7 +121,7 @@ export default class SchemaVersionForm extends Component {
   }
 
   handleSave() {
-    let {schemaText, description} = this.state;
+    let {schemaText, description, canonicalCheck} = this.state;
```
Comment (raju-saravanan) [inline webservice/src/main/resources/app/scripts/containers/Registry-Services/SchemaRegistry/SchemaVersionForm.jsx]: This should be called disableCanonicalCheck

File: webservice/src/main/resources/app/scripts/containers/Registry-Services/SchemaRegistry/SchemaVersionForm.jsx [hunk 3]
```diff
@@ -158,7 +164,7 @@ export default class SchemaVersionForm extends Component {
       gutters: this.props.schemaObj.type.toLowerCase() == 'avro' ? ["CodeMirror-lint-markers"] : [],
       lint: this.props.schemaObj.type.toLowerCase() == 'avro'
     };
-    let {schemaText, showError, changedFields, showCodemirror, schemaTextCompatibility} = this.state;
+    let {schemaText, showError, changedFields, showCodemirror, schemaTextCompatibility, canonicalCheck} = this.state;
```
Comment (raju-saravanan) [inline webservice/src/main/resources/app/scripts/containers/Registry-Services/SchemaRegistry/SchemaVersionForm.jsx]: This should be called disableCanonicalCheck disableCanonicalCheck

File: webservice/src/main/resources/app/scripts/containers/Registry-Services/SchemaRegistry/SchemaVersionForm.jsx [hunk 4]
```diff
@@ -169,6 +175,11 @@ export default class SchemaVersionForm extends Component {
               : "form-control"} value={this.state.description} required={true}/>
           </div>
         </div>
+        <div className="form-group">
+          <div className="checkbox">
+            <label><input name="canonicalCheck" onChange={this.handleToggleCanonicalCheck.bind(this)} type="checkbox" value={canonicalCheck} checked={canonicalCheck}/> Disable Canonical Check</label>
+          </div>
```
Comment (raju-saravanan) [inline webservice/src/main/resources/app/scripts/containers/Registry-Services/SchemaRegistry/SchemaVersionForm.jsx:181]: Same as the previous comment.

**Review by raju-saravanan — APPROVED** [review]
Overall LGTM, left some minor comments.

**Comment by joylyn** [comment]
Addressed the review comments.

**Pull request merged**